### DISCUSSION TITLE
Since the last update, I cannot export files

### DISCUSSION THREAD

**Discussion by mcallbee** [keiwando/toonsquid#446] [General]
Since the last update a day ago, when I try to export files, the software starts acting crazy and places the picture to be exported over the email recipient and subject lines so that you cannot see them.

**Comment by mcallbee** [comment]
![20231106_161955](https://github.com/keiwando/toonsquid/assets/138755339/26b166e8-6c5c-485e-99b7-b59f334bf722)
![20231106_162011](https://github.com/keiwando/toonsquid/assets/138755339/66b49b70-eba1-4506-8ff5-68d997552913)

**Comment by keiwando** [comment]
Hi! 
ToonSquid itself does not have a built-in e-mail client. The picture looks like the Gmail client, which ToonSquid has no control over. My first guess is that this is either a bug in Gmail or in iOS. I don't know which version of iOS you have installed but I've run into quite a lot of bugs in the 17.1 update so far.
For my understanding: Which "update" are you talking about? Did you update the ToonSquid app yesterday or the operating system of your iPad?

**Comment by mcallbee** [comment]
Thanks for getting back to me so quickly. The update that I am referring to
was a toonsquid update that my ipad alerted me to a few days ago. You are
correct, I also updated my ipad to  IOS 17.1 about the same time. Not sure
what to do, I really have to get these images off my ipad so that I can
put them into my word processor, which is on my desktop. Any suggestions??
I have no idea what caused it, but I thought by letting you guys know, it
could be fixed if it were on your end.

Thanks again,
I love the software. I don't use it for animation. I use it entirely for
drawing.

**Reply by kavlor** [reply]
Use Icloud or Dropbox.

**Comment by mcallbee** [comment]
Thank you for the suggestion...I ended up just using airdrop, it works
really well.
Thanks again

**Comment by mcallbee** [comment]
Apple had another bug fix update today but it did not fix it.

The action does not seem to be the email server...that info is all there
and correct, but I can only see it for a few seconds. It seems like
toonsquid is moving the image up, so that I cannot see the email
information. I am blind. Before, I could not see the toonsquid image, it
stayed down below until I was able to choose the email recipient. Then I
could hit send. Are others complaining of the same thing?

**Reply by keiwando** [reply]
I have not heard any other complaints about this yet. 
The ToonSquid app has absolutely no control over that e-mail window and how anything inside of it is displayed. It would actually be a huge privacy issue if other apps had direct access to your e-mail client like that. What happens during the export is that ToonSquid will export a temporary file to disk and then tell iOS "Here's the exported file. Please let the user choose what they want to do with it". From here on out, everything is handled by the operating system and the app that you select in that share popup. ToonSquid doesn't even know that you chose to share that file via e-mail.

**Reply by Byrdlnd** [reply]
These ppl complaining about the silliness,it's an anime software not Adobe 😒
Give this man some love ❤️ 
And ask for updates about things that really matter!!!!
Or go and create your own app I feel some of you want a million dollar app
But are u willing to pay for that?

**Comment by mcallbee** [comment]
I don't know what you are talking about. I have Procreate, Fresco and
Photoshop and I still use Toonsquid as my go to...that's not enough love? I
thought he and I were having a nice chat. I only put that issue out there
so that, if it is on the software side, it can be addressed quickly. Anyone
who thinks they can discern "attitude" from an email should not be in
forums.
